src/lib/customer.ts

```diff
@@ -1,6 +1,6 @@
 const CIO_BASE_URL = `https://beta-api.customer.io/v1/api/`
 
-async function fetchCustomer(cioId: string) {
+async function fetchCustomer(cioId: string, timeout: number = 600) {
```

Comment:
reduced this timeout since I added loading the auth'd user from egghead as well so the total time is under the 1.5s limit